### PR TITLE
add new field to asset

### PR DESCRIPTION
**Pull request by NNagato**

### CODE REVIEW

File: apidocs/source/includes/settings/_update_asset.md [hunk 1]
```diff
@@ -53,7 +53,8 @@ curl -X POST "https://gateway.local/v3/setting-change-main" \
                 "bid_spread": 0.1,
                 "single_feed_max_spread": 0.4,
                 "multiple_feeds_max_diff": 0.6
-            }
+            },
+            "normal_update_per_period": 1.234
```
Comment (secmask) [inline apidocs/source/includes/settings/_update_asset.md:57]: I thought this field is integer? please confirm with Luke

Comment (NNagato) [reply apidocs/source/includes/settings/_update_asset.md:57]: > I thought this field is integer? please confirm with Luke

he told me to use float for this field

File: reservesetting/storage/postgres/sql_schema.go [hunk 2]
```diff
@@ -65,6 +65,8 @@ CREATE TABLE IF NOT EXISTS "assets"
     
     normal_update_per_period FLOAT DEFAULT 1 
     CONSTRAINT normal_update_per_period_check CHECK(normal_update_per_period > 0),
+    max_imbalance_ratio FLOAT DEFAULT 2
+    CONSTRAINT max_imbalance_ratio_check CHECK(max_imbalance_ratio > 0),
```
Comment (secmask) [inline reservesetting/storage/postgres/sql_schema.go:69]: I thought migrate script has already does this?

Comment (NNagato) [reply reservesetting/storage/postgres/sql_schema.go:69]: yes, but we still need this for testing

File: reservesetting/storage/postgres/assets.go [hunk 3]
```diff
@@ -293,7 +297,10 @@ func (s *Storage) createAsset(
 		addressParam = &addressHex
 	}
 	if normalUpdatePerPeriod == 0 {
-		normalUpdatePerPeriod = 1 // set to default value
+		normalUpdatePerPeriod = defaultNormalUpdatePerPeriod
```
[the remark this answers is not in the document]
Comment (NNagato) [reply reservesetting/storage/postgres/assets.go]: fixed in https://github.com/KyberNetwork/reserve-data/pull/1003/commits/7ac9e9eae37a9f576fa707892f9485519e8da818

File: cmd/migrations/000006_add_new_columns_to_assets.up.sql [hunk 4]
```diff
@@ -0,0 +1,5 @@
+ALTER TABLE "assets" 
+ADD COLUMN IF NOT EXISTS normal_update_per_period FLOAT DEFAULT 1
```
[the remark this answers is not in the document]
Comment (NNagato) [reply cmd/migrations/000006_add_new_columns_to_assets.up.sql]: fixed in https://github.com/KyberNetwork/reserve-data/pull/1003/commits/7ac9e9eae37a9f576fa707892f9485519e8da818

**Comment by secmask** [comment]
there's a bit messing with migration scripts, but we can resolve it in a different task

**Comment by secmask** [comment]
@edwardEvans094 @ductm54 we have new field in create asset, please update your dashboard and I will deploy this to staging.

**Pull request merged**